contrib/grpcplugins/action/plugin-artifactory-release-bundle-distribute/main.go

```diff
@@ -98,11 +99,61 @@ func (actPlugin *artifactoryReleaseBundleDistributePlugin) Run(ctx context.Conte
 		return actionplugin.Fail("unable to distribute version: %v", err)
 	}
 
+	runResult, err := grpcplugins.GetRunResults(actPlugin.HTTPPort)
+	if err != nil {
+		return actionplugin.Fail("unable to list run results: %v", err)
+	}
+
+	fmt.Printf("Found %d run results\n", len(runResult))
+
+	repoToReindex := make(map[string]string, 0)
+	for _, r := range runResult {
+		// static-file type does not need to be released
+		if r.Type == sdk.WorkflowRunResultTypeStaticFile {
+			continue
+		}
+		rData, err := r.GetArtifactManager()
+		if err != nil {
+			return actionplugin.Fail("unable to read result %s: %v", r.ID, err)
+		}
+
+		repoToReindex[fmt.Sprintf("%s-release", rData.RepoName)] = rData.RepoType
+	}
+
+	for k, v := range repoToReindex {
+		// Filter on Helm repo
+		if v == "helm" {
+			fmt.Printf("%s reindex will start soon\n", k)
+			if err := actPlugin.reindex(distriClient, v, k); err != nil {
+				return actionplugin.Fail("unable to reindex: %v", err)
+			}
+		}
+	}
+
 	return &actionplugin.ActionResult{
 		Status: sdk.StatusSuccess,
 	}, nil
 }
 
+func (actPlugin *artifactoryReleaseBundleDistributePlugin) reindex(client art.DistribClient, repoType, repoKey string) error {
```

Comment:
Can this code be in intengrations/artifactory/artifactory.go to be use by others plugins too ?